drivers/storage/ide/atapi/data.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * PROJECT:     ReactOS ATA Port Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Global pageable data
+ * COPYRIGHT:   Copyright 2024 Dmitry Borisov <di.sean@protonmail.com>
+ */
+
+/* INCLUDES *******************************************************************/
+
+#include "atapi.h"
+
+/* GLOBALS ********************************************************************/
+
+const WCHAR* const AtapTargetToDeviceTypeKey[] =
+{
+    L"MasterDeviceType", L"SlaveDeviceType",
+#if defined(_M_IX86)
+    L"MasterDeviceType2", L"SlaveDeviceType2"
+#endif
+};
+
+static
+const ATAPORT_DEVICE_NAME AtapGenericDeviceNames[] =
+{
+    /* Device Type Generic Type     Peripheral ID             */
+    {"Disk",       "GenDisk",       "DiskPeripheral"           }, // DIRECT_ACCESS_DEVICE
+    {"Sequential", "GenSequential", "TapePeripheral"           }, // SEQUENTIAL_ACCESS_DEVICE
+    {"Printer",    "GenPrinter",    "PrinterPeripheral"        }, // PRINTER_DEVICE
+    /* There is no 'ProcessorPeripheral' */
+    {"Processor",  "GenProcessor",  "OtherPeripheral"          }, // PROCESSOR_DEVICE
+    {"Worm",       "GenWorm",       "WormPeripheral"           }, // WRITE_ONCE_READ_MULTIPLE_DEVICE
+    {"CdRom",      "GenCdRom",      "CdRomPeripheral"          }, // READ_ONLY_DIRECT_ACCESS_DEVICE
+    {"Scanner",    "GenScanner",    "ScannerPeripheral"        }, // SCANNER_DEVICE
+    {"Optical",    "GenOptical",    "OpticalDiskPeripheral"    }, // OPTICAL_DEVICE
+    {"Changer",    "GenChanger",    "MediumChangerPeripheral"  }, // MEDIUM_CHANGER
+    {"Net",        "GenNet",        "CommunicationsPeripheral" }, // COMMUNICATION_DEVICE,
+    {"Other",      "GenOther",      "OtherPeripheral"          }, // 10
+};
+
+const CHAR* const AtapBrokenInquiryDrive[] =
+{
+    "THOMSON-DVD",
+    "PHILIPS XBOX DVD DRIVE",
+    "PHILIPS J5 3235C",
+    "SAMSUNG DVD-ROM SDG-605B"
+};
```

Comment:
```suggestion
const CHAR* const AtapBrokenInquiryDrive[] =
{
    "THOMSON-DVD",
    "PHILIPS XBOX DVD DRIVE",
    "PHILIPS J5 3235C",
    "SAMSUNG DVD-ROM SDG-605B",
    /* TODO: Xbox 360 drives are also affected. */
};
```
Technically it's possible to connect Xbox DVD drives to an ordinary PC.

Regarding X360 drives, it was mentioned somewhere in XboxDev.

Reply:
I failed to find anything useful in the wiki. Can you list the X360 drives here please?

Reply:
It can be postponed then, since I have no information for these.